src/components/Loader/PeopleTable/PersonItem.tsx

```diff
@@ -0,0 +1,55 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../../types';
+import classNames from 'classnames';
+
+export const PersonItem = ({ person }: { person: Person }) => {
+  const { slug } = useParams();
```

Comment:
I`m not sure about it, but I think slug is not understandable for future developers, maybe we should create a variable?